dist/ExtUtils-ParseXS/t/001-basic.t

```diff
@@ -1607,6 +1607,13 @@ EOF
             [ 1, 0, qr{\QError: length() on non-parameter 's'\E.*line 6},
                    "got expected error" ],
         ],
+
+		[
+			'length of int is invalid',
+			['int', 'foo(int a, size_t length(a)'],
+			[ 1, 1 , qr/length\(NAME\) not supported with typemaps other than T_PV/, 'Got expected error about length' ],
+		],
+
```

Comment:
This doesn't look quite right. In the '[ 1, 1, ...]', the first 1 means check stderr rather than stdout, and the second means reverse the expectation, i.e. the test passes if the pattern _doesn't_ match (that's the bit which doesn't look right).

Reply:
You're right. That should be fixed now.